### PR TITLE
Tina studio2019

### PR DESCRIPTION
**Pull request by TinaCaputa**
Add single file projects based on the status selection
Update plugin version.
Add new unit tests.
Refactor UT: use Theory and InlineData.

### CODE REVIEW

File: Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ReportServiceUnitTests.cs [hunk 1]
```diff
@@ -34,41 +34,44 @@ private void GetReportPath_IsSuccessfullyReturned_UnitTest()
...
-		private void GetReportPath_IsNotSuccessfullyReturned_UnitTest()
+		[Theory]
+		[InlineData("")]
+		private void GetReportPath_IsNotSuccessfullyReturned_UnitTest(string filePath)
 		{
 			_reportService.SaveExportPath(_exportTempFolder);
-			var jsonPath = _reportService.GetJsonReportPath(string.Empty);
+			var jsonPath = _reportService.GetJsonReportPath(filePath);
 
 			Assert.False(!string.IsNullOrEmpty(jsonPath));
```
Comment (ealbu) [inline Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ReportServiceUnitTests.cs:44]: Just an observation:
  Here you assert that it is false that the string is not null; this is a double negation, much easier to read if you just make it positive like Assert.True(string.IsNullOrEmpty(jsonPath))

File: Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ProjectServiceUnitTests.cs [hunk 2]
```diff
@@ -62,12 +64,13 @@ private void RemoveSingleFileLanguage_IsSuccessfullyRemoved_UnitTest()
...
 		}
 
-		[Fact]
-		private void RemoveSingleFileLanguage_IsNotSuccessfullyRemoved_UnitTest()
+		[Theory]
+		[InlineData("English (United Kingdom)")]
+		private void RemoveSingleFileLanguage_IsNotSuccessfullyRemoved_UnitTest(string languageName)
 		{
 			// Arrange
 			var languagesToRemove = _exportReportsConfiguration.AddProjectLanguage();
```
Comment (ealbu) [inline Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ProjectServiceUnitTests.cs:72]: Same with the ""Get" instead of "Add""

Comment (TinaCaputa) [reply Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ProjectServiceUnitTests.cs:72]: I've updated

File: Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ReportServiceUnitTests.cs (hunk 1 above)
Comment (TinaCaputa) [reply Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ReportServiceUnitTests.cs:44]: I've updated

File: Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ProjectServiceUnitTests.cs [hunk 3]
```diff
@@ -48,12 +49,13 @@ private void AddFiles_IsNotSuccessfullyAdded_UnitTest()
...
 		}
 
-		[Fact]
-		private void RemoveSingleFileLanguage_IsSuccessfullyRemoved_UnitTest()
+		[Theory]
+		[InlineData("English (United Kingdom)")]
+		private void RemoveSingleFileLanguage_IsSuccessfullyRemoved_UnitTest(string languageName)
 		{
 			// Arrange
 			var languagesToRemove = _exportReportsConfiguration.AddMultipleLanguages();
```
[the remark this answers is not in the document]
Comment (TinaCaputa) [reply Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ProjectServiceUnitTests.cs:57]: I've updated the method. I prefere to leave the word Multiple for other scenarios when more languages will be added

**Pull request merged**